### PR TITLE
New deployment config

### PR DESCRIPTION
**Pull request by piobab**

### CODE REVIEW

**Comment by piobab** [comment]
This PR should be merged before: https://github.com/mars-protocol/red-bank/pull/264

File: scripts/deploy/base/deployer.ts [hunk 1]
```diff
@@ -217,6 +221,40 @@ export class Deployer {
     printYellow(`${assetConfig.symbol} updated.`)
   }
 
+  async initializeMarket(assetConfig: AssetConfig) {
```
Comment (brimigs) [inline scripts/deploy/base/deployer.ts:224]: I thought `init_asset` was taken away with params contract? Assets should just be added to params, no red bank execute msg needed?

File: scripts/deploy/osmosis/testnetConfig.ts [hunk 2]
```diff
@@ -55,14 +62,49 @@ export const atomAsset: AssetConfig = {
...
+  reserve_factor: '0.2',
+  interest_rate_model: {
+    optimal_utilization_rate: '0.8',
+    base: '0',
+    slope_1: '0.2',
+    slope_2: '2',
+  },
 }
 
+export const ionAsset: AssetConfig = {
```
Comment (brimigs) [inline scripts/deploy/osmosis/testnetConfig.ts:74]: ion can be deleted

File: scripts/deploy/osmosis/testnetConfig.ts [hunk 3]
```diff
@@ -77,52 +119,126 @@ export const USDCAsset: AssetConfig = {
...
     amount: '1000000000',
   },
   liquidation_threshold: '0.65',
   max_loan_to_value: '0.63',
   whitelisted: true,
 }
 
+export const atomOsmoVault: VaultConfig = {
+  addr: 'osmo1m45ap4rq4m2mfjkcqu9ks9mxmyx2hvx0cdca9sjmrg46q7lghzqqhxxup5',
+  symbol: 'ionOsmoVault',
```
Comment (brimigs) [inline scripts/deploy/osmosis/testnetConfig.ts:157]: symbol should be `atomOsmoVault`

**Review by brimigs — APPROVED** [review]

**Pull request merged**